inst/rmd/h/default.html

```diff
@@ -677,7 +677,7 @@ <h2 id="$idprefix$toc-title">$toc-title$</h2>
       hashGenerator: function (text) {
         return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
       },
-      ignoreSelector: ".toc-ignore",
+      ignoreSelector: ".toc-ignore, .unlisted.unnumbered",
```

Comment:
You can just ignore `.toc-ignore` with the change you did above ? 

(Which is BTW a clever one and indeed more scoped than by lua ! Thanks)

Reply:
We need to ignore `.unlisted.unnumbered` when `section_divs = false`.

Reply:
Sorry I mistyped the first time. I mean is this still needed ?

Reply:
Oh I see. That is why I found all those JS trick difficult to understand later on. We need to handle special cases in several places all the time. 
That means 

* section_divs = FALSE => we will have .unlisted.unumbered on all headers and no .toc-ignore
* section_divs = TRUE => we sill have .toc-ignore on all header and .unlisted .unnumbered in the sections divs above. 

All this does not help for future us !

Reply:
Moreover because we loose `.unlisted .unnumbered` on headers because we remove it ourselves in `header-attr.js` for newer Pandoc. 🤕

Reply:
OK, I simplified the trick by adding `.toc-ignore` to `.unlisted.unnumbered` with jQuery.
What do you think?

Reply:
A limitation is when section div has `.unlisted.unnumbered.toc-ignore` prior to the `tocify.js`.
In this case, `.toc-ignore` becomes duplicated, and the one remains in the final DOM.

Reply:
If we allow more complex selector, i.e. `.unlisted.unnumbered:not(.toc-ignore)`, we can avoid the limitation.

Reply:
I was thinking about the cleanest way and that may be just commenting to explain more (about this section divs trick)

But this last solution is easier to understand I think! Also the diff is really minimal as it is just one line addition! We will add `.toc-ignore` to all headers in the HTML though instead of handling directly the classes as option in your first solution. 

Sorry for all the thinking, I like to search for the different solution to a problem. 😉 

This is fine by me. Let's see what Yihui thinks about this.

Reply:
Great. Thanks for your deep thoughts. It really helped me to come up with this solution. I like it too!